### PR TITLE
Added auth to navbar and some minor changes

### PR DESCRIPTION
**Pull request by raman1999**

### CODE REVIEW

File: src/Components/Wishlist/WishItem.jsx [hunk 1]
```diff
@@ -22,10 +22,14 @@ export default function WishItem({ wishItem, userDispatch }) {
...
         </span>
       </p>
-      <span className="rating-block theme-shade-2">
-        {rating}
-        <i className="fa fa-star fa-md rating-checked"></i>
-      </span>
+      <div>
+        <span
+          className="rating-block txt-white"
+          style={{ backgroundColor: rating >= 3 ? "#26a541" : "#ff6161" }}
```
Comment (rahulcs754) [inline src/Components/Wishlist/WishItem.jsx]: My suggestion, you can create a variable before return and call here for more readability.

**Review by rahulcs754 — APPROVED** [review]
Just small suggestion given in this code and also use prettier to remove extra space 
other code is good and clean

**Review by SimranjitKaurMaan — APPROVED** [review]

File: src/Components/Wishlist/WishItem.jsx (hunk 1 above)
Comment (raman1999) [reply src/Components/Wishlist/WishItem.jsx]: Okay, I will change it

**Review by arundhatij12 — COMMENTED** [review]
looks good to merge

**Pull request merged**